src/lib/datepicker/datepicker.ts

```diff
@@ -226,6 +229,11 @@ export class MdDatepicker<D> implements OnDestroy {
     if (this._calendarPortal && this._calendarPortal.isAttached) {
       this._calendarPortal.detach();
     }
+    if (this._focusedElementBeforeOpen && 'focus' in this._focusedElementBeforeOpen) {
```

Comment:
It's there for IE's sake. In some cases it assigns the `activeElement` to something that doesn't have a `focus` method. It's what was throwing that 4mb error in the CI.